feat(policy): require human-written comments on github and zulip (#828)
leanprover-community/leanprover-community.github.io: templates/contribute/index.md

## Use of AI

> Note that comments on github and zulip must be hand-written; using AI for them is not allowed.

Author: Idea to get around the point from the discussion:
```suggestion
Note that using an LLM when writing comments on github or Zulip is not allowed.
```

Author: ```suggestion
Note that using an LLM when writing comments on github or Zulip is not allowed: use your own words.
```
